packages/core/src/types/plugin.ts

```diff
@@ -31,14 +97,13 @@ export interface RsdoctorWebpackPluginOptions<
     | Array<keyof Plugin.RsdoctorWebpackPluginFeatures>;
 
   /**
+   * @deprecated Now need to use output.mode.
```

Comment:
```suggestion
   * @deprecated Use `output.mode` instead, if you're using `lite` mode, please xxx
```

should give some executable suggestions to user, especially for `lite`

Reply:
DONE~